hash/apple2_flop_orig.xml

```diff
@@ -14272,7 +14272,7 @@ license:CC0
 	</software>
 
 	<software name="srccly22">
-		<description>The Sorcerer of Claymorgue Castle (Version 2.2/122)</description>
+		<description>Sorcerer of Claymorgue Castle (Version 2.2/122)</description>
```

Comment:
I changed this intentionally to make it consistent between the original and crack software lists, since it’s the same software.  The box art and the title screen are inconsistent.  The box art and other printed media says “The Sorcerer of Claymorgue Castle” (with article) while the title screen says “Sorcerer of Claymorgue Castle” (without article).  The same is true for ports, e.g. the DOS version.  Please choose one or the other – we can’t have the same game with different descriptions in different software lists.

Reply:
Corrected in  #7182